taggit_serializer/serializers.py

```diff
@@ -30,7 +30,12 @@ def update(self, object, validated_data):
     def _save_tags(self, tag_object, tags):
         for key in tags.keys():
             tag_values = tags.get(key)
-            [getattr(tag_object, key).add(tag) for tag in tag_values]
+            for tag in tag_values:
```

Comment:
Interesting approach, I will create an issue for it as an enhancement, thanks for pointing it out!